src/gql/generated/types.ts

```diff
@@ -186,12 +187,18 @@ export type QueryBuildVariantsForTaskNameArgs = {
   taskName: Scalars["String"];
 };
 
+export type QueryProjectSettingsArgs = {
+  identifier: Scalars["String"];
+};
+
 export type Mutation = {
   addFavoriteProject: Project;
   removeFavoriteProject: Project;
   schedulePatch: Patch;
   schedulePatchTasks?: Maybe<Scalars["String"]>;
   unschedulePatchTasks?: Maybe<Scalars["String"]>;
+  restartVersions?: Maybe<Scalars["String"]>;
+  /** @deprecated Field no longer supported */
```

Comment:
Cool was this automatically generated? I wasn't aware codegen did that for us.

Reply:
yeah! :)